# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../iputils/make_install.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2006-06-21 22:23:29.000000000 +0200
+++ ./Makefile	2006-06-21 22:25:07.000000000 +0200
@@ -88,3 +88,5 @@
 	@make man
 	@cd ..; tar c iputils | gzip -9c > `date +iputils-ss%y%m%d.tar.gz`
 
+install: all
+	cp -v $(TARGETS) $(DESTDIR)$(SBINDIR)
